modules/migration-obtaining-service-account-token-camt.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
```

Comment:
I don't see this file included in the assembly anywhere, should it be? Or was this step included into the "Adding a cluster to the CAM web console" procedure and this file should be removed?

Reply:
Good catch. I've removed the file.